website/settings/defaults.py

```diff
@@ -1832,3 +1832,19 @@ def parent_dir(path):
 }
 
 PREPRINTS_ASSETS = '/static/img/preprints_assets/'
+
+SOCIAL_FIELDS = {
```

Comment:
I don't think it is necessary. I was having a little trouble with the importing OSFUser into the validator. I'm sure I could have worked around that trouble, but when I saw what I took to be hard coded configuration in OSFUser, I figured it belonged in a config file. This is the first PR I've done working with django models, so not sure what's best practice.
  Want I should move it back into OSFUser?